src/components/ActionMenuList/index.js

```diff
@@ -10,13 +10,15 @@ import DeleteMessageIcon from '../../assets/images/DeleteMessageIcon.svg';
 import CloseIcon from '../../assets/images/CloseIcon.svg';
 import MarkIcon from '../../assets/images/Mark.svg';
 import UnmarkIcon from '../../assets/images/Unmark.svg';
+import PreviewIcon from '../../assets/images/Preview.svg';
+import DownloadIcon from '../../assets/images/Download.svg';
 
 import dynamicsFont from '../../assets/DynamicsFont/DynamicsFont.scss';
 
 import styles from './styles.scss';
 import i18n from './i18n';
 
-function ConfirmDeleteModal({
+export function ConfirmDeleteModal({
```

Comment:
Why export this function?

Reply:
used in tests, can be separated into independent files when needed